vm/src/types.rs

```diff
@@ -9,36 +9,82 @@ pub type VMIndex = u32;
 pub type VMTag = u32;
 pub type VMInt = isize;
 
+/// Enum which represent the instructions executed by the virtual machine.
+///
+/// The binary arithmetic instructions pop two values of the stack and pushes the result of
+/// adding/comparing/etc the values. 
```

Comment:
'The binary arithmetic instructions pop two values of the stack and then push the result.'